src/Apps/Artwork/Components/OtherWorks/RelatedWorksArtworkGrid.tsx

```diff
@@ -51,6 +51,7 @@ class RelatedWorksArtworkGrid extends React.Component<
     this.props.relay.refetch(
       {
         layerId: tab.data.layerId,
+        artworkSlug: this.props.artwork.id,
```

Comment:
Unrelated to this, but why does this take a slug instead of an ID?

Reply:
Good call, this could change and just be `artworkID`, and the variable in the refetch query can change too.

I think since most of the time these components only have access to the slug (and not the id), taken from the URL, that's generally the variable being used.

But in this client-side refetch, can totally be `artworkID` instead (if indeed that is better).